honggfuzz.BUILD

```diff
@@ -0,0 +1,176 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Bazel rules for building the Honggfuzz binary and the library linked with the
+# fuzz test executables.
+#
+# To use Honggfuzz, the following OS packages need to be installed:
+#   * libunwind-dev
+#   * libblocksruntime-dev
+#
+# NOTE: Currently, Honggfuzz is unable to detect buffer overflows in the input
+# string passed to the fuzz test entry point. This limitation is *not* caused
+# by the build options below limiting the use of sanitizers in the Honggfuzz
+# libraries.
+
+load("@rules_cc//cc:defs.bzl", "cc_binary", "cc_library")
+
+COMMON_COPTS = [
+    "-D_GNU_SOURCE",
+    "-D_HF_ARCH_LINUX",
+    "-fPIC",
+    "-Wall",
+    "-Wextra",
+    "-Werror",
+    "-Wno-override-init",
+    "-Wno-initializer-overrides",
+    "-Wno-gnu-empty-initializer",
+    "-Wno-format-pedantic",
+    "-Wno-gnu-statement-expression",
+    "-mllvm",
+    "-inline-threshold=2000",
+    "-fblocks",
+
+    # Do not instrument Honggfuzz itself, in order to avoid recursive
+    # instrumentation calls that would crash the fuzz test binary.
+    "-fsanitize-coverage=0",
+    "-fno-sanitize=address,thread,undefined",
```

Comment:
any others like memory need to be added too ?

Reply:
Good point. I've just discovered that Clang accepts `-fno-sanitize=all`, so that should cover all current and future sanitizer configurations.